pkg/nodebootstrap/legacy/ubuntu.go

```diff
@@ -59,7 +60,13 @@ func (b UbuntuBootstrapper) UserData() (string, error) {
 }
 
 func makeUbuntuConfig(spec *api.ClusterConfig, ng *api.NodeGroup) ([]configFile, error) {
-	clientConfigData, err := makeClientConfigData(spec, kubeconfig.HeptioAuthenticatorAWS)
+	supportsAWSEKS, err := utils.IsMinVersion(api.Version1_20, spec.Metadata.Version)
```

Comment:
```suggestion
	supportsAWSEKS, err := utils.IsMinVersion(api.Version1_20, spec.Metadata.Version)
	if err != nil {
		return nil, err
	}
```
needs an error check here